projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -508,6 +508,7 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy, ColumnTy
         }
     }
 
+    /** @hidden @internal **/
```

Comment:
We have a section for that actually - https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/column-resizing#auto-size-columns-on-double-click